src/universal/types/constEnums.d.ts

```diff
@@ -8,3 +8,9 @@ export const enum LoaderSize {
   WHOLE_PAGE = 400,
   PANEL = 200
 }
+
+// equal to utils/constants
```

Comment:
why assert the equality with a comment instead of importing `utils/constants` and using the symbols directly to define the values of the enum instead?

Reply:
a `const enum` requires a string or number for a value, no imports allowed.
i'm not 100% sure why this is a constraint.

they're such a nice feature though because at compile time we're left with just strings, no imports & no .d.ts file.